### PR TITLE
docs(readme): move monorepo contributing guide to contributing

### PR DESCRIPTION
**Pull request by kumikokashii**
This update goes together with frontend's readme & contributing update https://github.com/HospitalRun/hospitalrun-frontend/pull/2177

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -44,6 +53,8 @@ Note: All repos use [conventional commits](https://www.conventionalcommits.org)
 
 ## 5. Submit a Pull Request
 
+‼️ **Temporarily, please wait to open a Pull Request until it's ready for review. We need to optimize build time.**
```
Comment (jackcmeyer) [inline .github/CONTRIBUTING.md:56]: this line can get removed, we have fixed this issue :)

File: README.md [hunk 2]
```diff
@@ -26,7 +26,6 @@ Built with React, Node, PouchDB/CouchDB and using offline first design, we're wo
 - [Contributing](#contributing)
 - [Project Structure](#project-structure)
 - [Application Infrastructure](#application-infrastructure)
-- [Monorepo-specific Contributing Guide](#monorepo-specific-contributing-guide)
```
Comment (tehKapa) [inline README.md:29]: I'm not so sure I want to remove this chapter

Comment (kumikokashii) [reply README.md:29]: Do you mean, it might be better staying in README than moved to CONTRIBUTING?

Comment (tehKapa) [reply README.md:29]: Maybe not, never mind.

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (kumikokashii) [reply .github/CONTRIBUTING.md]: ✅

**Pull request merged**